Rawdata

LienReyserhove opened the pull request:
The following steps were carried out: 

1. Downloading the [original dataset](http://data.inbo.be/ipt/resource?r=alien-macroinvertebrate-occurrences&v=1.6) (`alien_macroinvertebrates.txt`) and changing this from `.txt` to `.tsv`

2. Reducing this original dataset in OpenRefine (cleaning), to retain only the columns containing the essential information. The following columns were removed:

- [x] type
- [x] license
- [x] rightsHolder
- [x] accessRights
- [x] bibliographicCitation
- [x] datasetID
- [x] InstitutionCode
- [x] datasetName
- [x] ownerInstitutionCode
- [x] basisOfRecord
- [ ] verbatimEventdate
- [x] continent
- [x] countryCode
- [ ] verbatimCoordinates
- [x] verbatimSRS
- [x] geodeticDatum
- [x] kingdom
- [x] verbatimTaxonRank
- [x] nomenclaturalCode

3. Replacing the original (downloaded) occurences dataset by the cleaned version.

4. Generating  .`Rproj` + basic code for reading the dataset in R.

LienReyserhove committed:
Original dataset

Importing the original dataset. The format was changed from .txt to .tsv

URL: http://data.inbo.be/ipt/resource?r=alien-macroinvertebrate-occurrences&v=1.6

LienReyserhove committed:
New R project

LienReyserhove committed:
Inlezen data + set locale

LienReyserhove committed:
Creëeren van basis occurences dataset in OpenRefine

Toevoegen van subspecies waarde in taxonRank, in overeenstemming met SubSpp in verbatimTaxonRank

Verwijderen van volgende kolommen:
- type
- license
- rightsHolder
-accessRights
-bibliographicCitation
-datasetID
-InstitutionCode
-datasetName
-ownerInstitutionCode
-basisOfRecord
-verbatimEventdate
-continent
-countryCode
-verbatimCoordinates
-verbatimSRS
-geodeticDatum
-kingdom
-verbatimTaxonRank
- nomenclaturalCode

LienReyserhove committed:
Update gitignore for R files

LienReyserhove committed:
Exclude Rproj rubbish from history

LienReyserhove committed:
Add mapping file dwc occ

LienReyserhove committed:
Remove redundant Rproj files

peterdesmet committed:
Use default R gitignore + add extra rules at bottom

peterdesmet reviewed (changes requested):
Please correct:

* [ ] Remove id (is same as occurrenceID)
* [ ] eventDate is no longer in ISO format (2001-05-06 vs 6/05/2001), maybe something happened in Google Refine?
* [ ] verbatimCoordinates should be kept (and split in x and y)
* [ ] decimalLatitude/longitude contains multiple dots (probably something happened on export)

Please describe:

* [ ] is verbatimEventDate exactly the same as eventDate? Would warrant to have it removed

LienReyserhove committed:
Reloading the raw alien_macroinvertebrate_occurences.tsv file

The new raw dataset based on suggestions of @peterdesmet

- id was removed
- eventDate is now again in ISO format, dots in coordinates were removed
- verbatimcoordinates were split (now verbatimLongitude and verbatimLatitude)
- decimalLatitude and decimalLongitude were retained (but see issue #14)
- verbatimEventDate is the same as eventDate and was thus removed
